build_ut.py

```diff
@@ -158,11 +160,13 @@ def getSuitesToRun (suitesRequested):
 # Build the target executables
 def buildTargets (suites, outputDir, resultsFile, debug, coverage):
     # Run cmake
-    cmakeCmd = ["cmake", "-B", outputDir , "-DCMAKE_BUILD_FLAG=UnitTests", "-DRIALTO_ENABLE_DECRYPT_BUFFER=1"]
-    if debug:
-        cmakeCmd.append("-DCMAKE_BUILD_TYPE=Debug")
-    if coverage:
-        cmakeCmd.append("-DCOVERAGE_ENABLED=1")
+    cmakeCmd = ["cmake", "-B", outputDir,"-DCMAKE_BUILD_FLAG=UnitTests", "-DRIALTO_ENABLE_DECRYPT_BUFFER=1" ]
+    # if debug:
+    cmakeCmd.append("-DCMAKE_BUILD_TYPE=Debug")
+    # if coverage:
```

Comment:
Tidy this up for delivery, shouldn't push commented code to master and need to re-enable coverage